### PR TITLE
Ir/#5/create db and table

### PR DESCRIPTION
**Pull request by nesyG**
Added simple models,created route and controller for populating db with mock data

### CODE REVIEW

**Comment by nesyG** [comment]
image

File: server/src/models/listing.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import mongoose, { Schema, Types } from "mongoose";
+
+export interface IListing {
+  user_id: Types.ObjectId;
+  listing_id: string
+  car_price: string;
+}
+
+const UsersSchema = new Schema<IListing>({
```
Comment (rradman) [inline server/src/models/listing.ts]: create a listing here.

File: server/src/models/listing.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  user_id:  { 
+    type: mongoose.Schema.Types.ObjectId, ref: "User", required: false
```
Comment (rradman) [inline server/src/models/listing.ts]: Can we import User here so we dont have to type "User"

File: server/src/models/car.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  car_model_year: {
+    type: String,
+    required: true,
+  },
+  car_color: {
+    type: String,
+    required: true,
+  },
+  user_id: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```
Comment (rradman) [inline server/src/models/car.ts]: Remove this. Use User ref on a Listing instead

**Review by rradman — CHANGES_REQUESTED** [review]

File: server/src/models/listing.ts (hunk 2 above)
Comment (nesyG) [reply server/src/models/listing.ts]: No

**Pull request merged**